mmv1/third_party/terraform/.teamcity/components/inputs/services_ga.kt

```diff
@@ -614,7 +614,12 @@ var ServicesListGa = mapOf(
     "securitycentermanagement" to mapOf(
         "name" to "securitycentermanagement",
         "displayName" to "Securitycentermanagement",
-        "path" to "./google/services/securitycentermanagement"
+        "path" to "./google-beta/services/securitycentermanagement"
```

Comment:
This change should be removed from this PR

Reply:
addressed

Reply:
Still unaddressed

Reply:
this service is added, because of merge conflicts from the main branch.